packages/typespec-vscode/test/manual/import-typespec-from-openapi3-test-plan.md

```diff
@@ -0,0 +1,86 @@
+# Import TypeSpec from OpenAPI3 Test Scenario _(stretch goal for SE)_
+
+With the TypeSpec emitter for OpenAPI3, users can import a TypeSpec file from a designated OpenAPI3 document. While it is possible to repeatedly convert OpenAPI3 to TypeSpec.
+
+## Test Environment
+
+- OS : Windows or Linux
+
+> Note: The extension should support all test cases in VS Code for Windows and Linux. Mac support is a stretch goal for Selenium semester.
+
+## Prerequisites
+
+Install TypeSpec Compiler before starting to write TypeSpec.
+
+- [Node.js 20+](https://nodejs.org/download/)
+- Npm 7+
+- [Install TypeSpec Compiler CLI](https://typespec.io/docs/): `"npm install -g @typespec/compiler"`
+
+## Test Steps
+
+### Step 1: Install the typespec extension.
+
+_Option 1_. Install using .vsix file:
+`Extension` -> `…` -> `Install form VSIX...`
+
+![alt text](./images/InstallTypespec_VSIX.png)
+
+Find the .vsix file you want to install locally.
+
+![alt text](./images/InstallTypeSpec_SelectVSIXFileTest.png)
+
+_Option 2_. Install typespec with vscode extension marketplace:
+`Extension` -> input `TypeSpec for VS Code` -> `Install`
+
+![alt text](./images/InstallTypespec_ExtensionMarketplaceTest01.png)
+
+### Step 2: "Import TypeSpec from OpenAPI 3.0" from the right-click context menu of a .tsp file.
+
+![alt text](./images/TriggerImportTypeSpecfromOpenAPI3.png)
+
+### Step 3: Confirm the project folder where you will place the TypeSpec file converted from the specified OpenAPI3 specification.
+
+![alt text](./images/ImportTypeSpecfromOpenAPI3_ConfirmProjectFolder.png)
+
+- Select a non-empty folder:
+
+  **Validate:** Will it appear: `The selected folder isn't empty. Do you want to continue? Some existing files may be overwritten.`
+
+  ![alt text](./images/ImportTypeSpecfromOpenAPI3_VerifyFolderIsEmpty.png)
+
+- Select a empty folder:
+
+  If the folder is empty, skip the query and go to the next step.
+
+### Step 4: Specify the OpenAPI3 specification to convert.
+
+![alt text](./images/ImportTypeSpecfromOpenAPI3_SpecifyOpenAPI3Specification.png)
+
+### Step 5: Verify that `@typespec/http` and `@typespec/openapi3` are installed.
```

Comment:
only openapi3 is needed and will be prompt if user didn't have it installed. Please check extension from PR #5451 for the latest user experience.